db/migrate/20230823160153_create_user_data.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserData < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user_data do |t|
+      t.decimal :annual_leave_remaining, precision: 3, scale: 1, null: false
```

Comment:
I think this is a good choice of data type for this column. Decimals are good for anything you might want to do maths with